crates/agglayer-node/src/lib.rs

```diff
@@ -64,6 +60,10 @@ pub fn main(
         .enable_all()
         .build()?;
 
+    // Initialize the tracing
+    metrics_runtime
+        .block_on(async { agglayer_telemetry::traces::setup_tracing(&config.log, version) })?;
```

Comment:
This isn't async, thus you can execute the `setup_tracing` at the same place as `logging::tracing`